### PR TITLE
Release 1.14.0

### PR DESCRIPTION
**Pull request by hiddeco**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,151 @@
...
+- `fluxctl` has a new `install` command to ease generating the YAML
+  manifests required to deploy Flux
+  [fluxcd/flux#2287][]
+- Kubectl and Kustomize have been upgraded
+  - `kubectl` -> `1.13.8` [fluxcd/flux#2327][]
+  - `kustomize` -> `3.1.0` [fluxcd/flux#2299][]
+- The annotation domain has been changed to `fluxcd.io`, but backwards
+  compatibility with the old (`flux.weave.works`) domain is maintained
+  [fluxcd/flux#2219][]
+- The amount of sorts done by `ListImagesWithOptions` has been reduced
```
Comment (squaremo) [inline CHANGELOG.md]: s/amount/number/

Comment (squaremo) [reply CHANGELOG.md]: (amount is usually used only for continuous quantities, e.g., "that is a large amount of water")

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,151 @@
...
+  [fluxcd/flux#2338][]
+- `fluxctl` will only look for running `fluxcd` pods while attempting
+  to setup a port forward
+  [fluxcd/flux#2283][]
+- `--registry-poll-interval` has been renamed to `--automation-interval`
+  to better reflect what it controls; the interval at which automated
+  workloads are checked for updates, and updated.
+  [fluxcd/flux#2284][]
+- `fluxctl` now has a global `--timeout` flag, which controls how long
+  it waits for jobs fired of to `fluxd` to complete
```
Comment (squaremo) [inline CHANGELOG.md]:
```suggestion
  it waits for jobs sent to `fluxd` to complete
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,151 @@
...
+    [#2315][fluxcd/flux#2315], [#2332][fluxcd/flux#2332], [#2351][fluxcd/flux#2351], [#2353][fluxcd/flux#2353],
+    [#2358][fluxcd/flux#2358], [#2363][fluxcd/flux#2363], [#2364][fluxcd/flux#2364], [#2365][fluxcd/flux#2365],
+    [#2367][fluxcd/flux#2367], [#2368][fluxcd/flux#2368], [#2372][fluxcd/flux#2372]}
+- Soon-to-be deprecated version script has been removed from the Snapcraft
+  build configuration
+  [fluxcd/flux#2350][]
+- Various typos have been fixed
+  fluxcd/flux{[#2348][fluxcd/flux#2348], [#2352][fluxcd/flux#2352], [#2295][fluxcd/flux#2295]}
+- Various CI build tweaks (i.a. support preleases containing numbers, Go
+  tarbal cleanup after installation, Helm chart release changes)
```
Comment (squaremo) [inline CHANGELOG.md]:
```suggestion
  tarball cleanup after installation, Helm chart release changes)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,151 @@
...
+[fluxcd/flux#2299]: https://github.com/fluxcd/flux/pull/2299
+[fluxcd/flux#2301]: https://github.com/fluxcd/flux/pull/2301
+[fluxcd/flux#2302]: https://github.com/fluxcd/flux/pull/2302
+[fluxcd/flux#2306]: https://github.com/fluxcd/flux/pull/2306
+[fluxcd/flux#2311]: https://github.com/fluxcd/flux/pull/2311
+[fluxcd/flux#2312]: https://github.com/fluxcd/flux/pull/2312
+[fluxcd/flux#2313]: https://github.com/fluxcd/flux/pull/2313
+[fluxcd/flux#2314]: https://github.com/fluxcd/flux/pull/2314
+[fluxcd/flux#2315]: https://github.com/fluxcd/flux/pull/2315
+[fluxcd/flux#2315]: https://github.com/fluxcd/flux/pull/2315
```
Comment (squaremo) [inline CHANGELOG.md]: listed twice

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,150 @@
...
+  initial clone
+  [fluxcd/flux#2371][]
+
+### Enhancements
+
+- With `--git-readonly`, `fluxd` can now sync a git repo without having
+  write access to it. In this mode, `fluxd` will not make any commits
+  to the repo.
+  [fluxcd/flux#1807][]
+- Mapping images in `HelmRelease resources` using YAML dot notation
```
Comment (squaremo) [inline CHANGELOG.md]: Say "dot notation annotation" ten times, fast :-)

Comment (hiddeco) [reply CHANGELOG.md]: It is my new favourite word combination.

Comment (squaremo) [reply CHANGELOG.md]: Then it is decided: henceforth we shall exploit all opportunities to use the term "dot notation annotation"

**Pull request merged**